static/app/components/replays/replayController.tsx

```diff
@@ -109,23 +110,52 @@ function ReplayCurrentTime() {
   );
 }
 
-function ReplayPlaybackSpeed({speedOptions}: {speedOptions: number[]}) {
-  const {setSpeed, speed} = useReplayContext();
+function ReplayOptionsMenu({speedOptions}: {speedOptions: number[]}) {
+  const {setSpeed, speed, isSkippingInactive, toggleSkipInactive} = useReplayContext();
+  const SKIP_OPTION_VALUE = 99999;
+
   return (
-    <CompactSelect
-      triggerProps={{
-        size: 'xs',
-        prefix: t('Speed'),
-      }}
-      value={speed}
-      options={speedOptions.map(speedOption => ({
-        value: speedOption,
-        label: `${speedOption}x`,
-        disabled: speedOption === speed,
-      }))}
-      onChange={opt => {
-        setSpeed(opt.value);
-      }}
+    <CompositeSelect
+      closeOnSelect
+      isDismissable
+      placement="bottom"
+      trigger={({props, ref}) => (
+        <Button
+          ref={ref}
+          {...props}
+          size="xs"
+          title={t('Settings')}
+          aria-label={t('Settings')}
+          icon={<IconSettings size="sm" />}
+        />
+      )}
+      sections={[
+        {
+          defaultValue: speed,
+          label: 'Playback Speed',
+          value: 'playback_speed',
+          onChange: setSpeed,
+          options: speedOptions.map(sp => ({
+            label: `${sp}x`,
+            value: sp,
+          })),
+        },
+        {
+          defaultValue: isSkippingInactive ? SKIP_OPTION_VALUE : undefined,
+          multiple: true,
+          label: '',
+          onChange: value => {
+            toggleSkipInactive(Array.isArray(value) && value.length > 0);
```

Comment:
it's weird to me that we're checking `isArray` here, but in the other section it's possible to only have `onChange: setSpeed,`

I've gotta read the docs to understand how this works and will come back after

Reply:
That list is set to multi so the value can toggle so it returns an array of values. Thats why I said it seemed a little bit hacky but it's functional without having to build out a new dropdown/select component

Reply:
I think if we add types to the function args that would help document what's expected: 

```
onChange: (value: typeof SKIP_OPTION_VALUE[]) => {
  toggleSkipInactive(value.length > 0);
```